### PR TITLE
Added prior based per-element-scale in LSTM recipe.

### PR DESCRIPTION
**Pull request by Minhua722**

### CODE REVIEW

File: egs/wsj/s5/steps/nnet3/lstm/make_configs.py [hunk 1]
```diff
@@ -82,8 +91,10 @@ def AddOutputNodeWithDelay(config_lines, input, label_delay):
     component_nodes = config_lines['component-nodes']
     component_nodes.append('output-node name=output input=Offset({0},{1})'.format(input['descriptor'], label_delay))
 
-def AddFinalLayer(config_lines, input, output_dim, ng_affine_options = ""):
+def AddFinalLayer(config_lines, input, output_dim, ng_affine_options = "", presoftmax_scale = ""):
```
Comment (vijayaditya) [inline egs/wsj/s5/steps/nnet3/lstm/make_configs.py]: Use presoftmax_scale default option as None.

**Comment by danpovey** [comment]
Can we please do some testing of this before merging?  I don't want to disrupt people's experiments with a totally unknown quantity.

**Comment by danpovey** [comment]
sorry did not mean to close.

**Comment by vijayaditya** [comment]
@Minhua722  could also modify the steps/nnet3/make_tdnn_configs.py to add the per-element-scale component.

**Comment by vijayaditya** [comment]
@Minhua722  Could you please pull the latest changes for the lstm recipe and retest.

**Comment by Minhua722** [comment]
@vijayaditya OK

**Pull request closed without merging**